src/main.c

```diff
@@ -26,6 +27,8 @@ static void init_default_values()
     output_video = TRUE;
 }
 
+char *submitter = NULL;
```

Comment:
Nit: because `submitter` is only used in this file, it could be marked as `static`.